plugins/cylance_protect/plugin.spec.yaml

```diff
@@ -24,22 +29,25 @@ key_features:
   - Quarantine endpoints
   - Search threats
   - Delete assets
+links:
+  - "[BlackBerry CylancePROTECT](https://www.cylance.com)"
 requirements:
   - CylancePROTECT configured with an Custom Application
 references:
-  - "[BlackBerry CylancePROTECT](https://www.cylance.com)"
+  - "[BlackBerry CylancePROTECT API](https://documentation.securonix.com/bundle/securonix-on-prem-user-guide/page/content/active-deployment-guides/cylance-protect-api-kvp.htm)"
```

Comment:
this doc reference doesn't seem correct

Reply:
The reference links to the API docs for Cylance and the link refers the Cylance's website.

Reply:
oh interesting i got a 404 when went to it, must have copied a bracket

Reply:
@rmurray-r7 as discussed, I think this should link to the Blackberry docs please.